.github/workflows/node.js.yml

```diff
@@ -23,11 +23,19 @@ jobs:
         node-version: [10.x, 12.x, 14.x]
 
```

Comment:
same question here!

Reply:
Can we have the selective running for PRs . 
And run all the jobs when we push into main ?